File-Formats/PDF/Working-with-Document-Conversions.md

```diff
@@ -1,11 +1,11 @@
 ---
-title: Working with Document Conversion | Syncfusion
-description: This section explains converting other document Types (Word, Excel, RTF, TIFF, XPS and HTML) into PDF
+title: Working with Document Conversion in File Formats PDF Control | Syncfusion
+description: Learn here about Working with Document Conversion in Syncfusion Essential File Formats PDF Control, its elements, and more.
 platform: file-formats
 control: PDF
 documentation: UG
 ---
-# Working with Document Conversions
+# Working with Document Conversions in Fileformat PDF 
```

Comment:
Change like this - Working with Document Conversions in File Formats PDF

Reply:
Resolved